src/main/java/org/fundacionjala/pivotal/api/APICommons.java

```diff
@@ -0,0 +1,48 @@
+package org.fundacionjala.pivotal.api;
+
+import com.jayway.restassured.builder.RequestSpecBuilder;
+import com.jayway.restassured.path.json.JsonPath;
+import com.jayway.restassured.response.Response;
+import com.jayway.restassured.specification.RequestSpecification;
+import org.fundacionjala.pivotal.common.PropertiesConfig;
+
+/**
+ * APICommons.java
+ * Class with common method to execute an API request.
+ */
+public final class APICommons {
+
+    /**
+     * Constructor.
+     */
+    private APICommons() {
+
+    }
+
+    /**
+     * Method to get an element from the response.
+     *
+     * @param res the response from request.
+     * @param element the element needed.
+     * @return the body of response.
+     */
+    public static String getElementResponse(final Response res, final String element) {
+        JsonPath jsonPathEvaluator = res.jsonPath();
+        return jsonPathEvaluator.get(element).toString();
+    }
+
+    /**
+     * Method to build request header.
+     *
+     * @return the request specification.
+     */
+    public static RequestSpecification buildRequestHeader() {
+        RequestSpecification requestSpecification = new RequestSpecBuilder()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "requestSpecification". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)